modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -1,3 +1,249 @@
+/datum/job/ai/New()
+	.=..()
+	alt_titles += list(
+		"Abominable Intelligence",
+		"Artificial Intelligence",
+		"Big Brother",
+		"Big Sibling",
+		"Big Sister",
+		"Machine Spirit",
+	)
+
+/datum/job/assistant/New()
+	.=..()
+	alt_titles += list(
+		"Bastard",
+		"Call Boy",
+		"Call Girl",
+		"Callperson",
+		"Crew Gremlin",
+		"Escort",
+		"Fiend",
+		"Fleshlight",
+		"Hooker",
+		"Jackanape",
+		"Maintworker",
+		"Ne'er-do-well",
+		"Prostitute",
+		"Roustabout",
+		"Sexworker",
+		"Slut",
+		"Spacefarer",
+		"Stripper",
+		"Troublemaker",
+		"Whore",
+	)
+//Finally, a name to represent the average Assistant.
+
+/datum/job/atmospheric_technician/New()
+	.=..()
+	alt_titles += list(
+		"Pipe Cleaner",
+		"Pipe Inspector",
+		"Pipe Technician",
+	)
+
+/datum/job/bitrunner/New()
+	.=..()
+	alt_titles += list(
+		"Netrunner",
+		"Matrix Traversal Technician",
+	)
+
+/datum/job/blueshield/New()
+	.=..()
+	alt_titles += list(
+		"Blueshit",
+		"Chief Steward",
+		"Command Enforcer",
+		"Command Goon",
+		"Command Henchman",
+		"Steward",
+	)
```

Comment:
absolutely not

Reply:
I have no idea what is even wrong with this so I took the top and bottom one out